llvm/lib/Transforms/Yk/ControlPoint.cpp

```diff
@@ -86,24 +81,25 @@ const unsigned CPShadow = 13;
 
 using namespace llvm;
 
-/// Find the call to the dummy control point that we want to patch.
-/// Returns either a pointer the call instruction, or `nullptr` if the call
-/// could not be found.
-/// YKFIXME: For now assumes there's only one control point.
-CallInst *findControlPointCall(Module &M) {
+/// Find the calls to the dummy control point that we want to patch.
+/// Returns either a pointer the a vector of instructions
```

Comment:
I don't understand the comment. Maybe a typo somewhere?

Reply:
Updated :point_right: f8508850c7fad09c6dec798d1c5e436bce38b9bf